source/request-as-event-emitter.js

```diff
@@ -32,7 +32,12 @@ module.exports = options => {
 			return;
 		}
 
-		let fn = options.protocol === 'https:' ? https : http;
+		let fn;
+		if (options.fn) {
+			fn = options.fn; // eslint-disable-line prefer-destructuring
```

Comment:
`({fn} = options);`

Reply:
Isn't it ugly? :P

Reply:
Yes... Ok, don't do it.

Reply:
Fixed: https://github.com/xojs/eslint-config-xo/commit/20daf6d7798140b05e690dfab63959e7c21bfed9 You can remove the disable comment now.